### PR TITLE
Wip/correct fonts and hp tweaks

### PR DESCRIPTION
**Pull request by luxumbra**
This tidies up the styling of the OfferHelp Wizard and places the prev/next nav items at the bottom of the page (I believe this was something that was wanted?). The font being used for the section headings is correct. I am still not able to get Work Sans installed and working on my build.

I still have the homepage buttons to sort out and the NeedHelp wizard needs the style changes applied. I shall copy the styled components from OfferHelp into their own files for better DRY, if you're happy with what I have done here. :)
image
image
image
image

### CODE REVIEW

**Comment by luxumbra** [comment]
@shivanshs9 @robinv85 looks like you guys have worked on the homepage / Offer help wizard. Would you be happy to review when you have time? :pray:

**Comment by robinv85** [comment]
Reviewing... @luxumbra can ou share a screenshot for the prev/next buttons?
Probably need to check with a designer but AFAIK, I think it was mostly the "submit" button that's kind of at the bottom of the page, but that is because the design looked like a mobile design, which I'm not sure is also feasible for a mobile website... seems like we are trying to make it look like an app.

**Comment by luxumbra** [comment]
Ah yes, sorry. I forgot to add the screen shots...kinda helps! :sweat_smile: 

I'll post in Design for feedback shortly.. Thanks @robinv85

**Comment by luxumbra** [comment]
I've noticed a bug in the email field where the input loses focus with every keypress and step 3 skips when moving forward through the steps. I'll sort that now.

File: client/src/components/Button/CustomButton.js [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
 import { Button } from "antd-mobile";
 import { ROYAL_BLUE, SELAGO } from "../../constants/colors";
-
-/* 
-  Props accepted: 
+import { theme } from "../../constants/theme";
+const { colors } = theme;
+/*
+  Props accepted:
   inline, primary, large, whitebg, roundborder
```
[the remark this answers is not in the document]
Comment (luxumbra) [reply client/src/components/Button/CustomButton.js:8]: Good idea @shivanshs9 , thanks.

File: client/src/components/Button/CustomButton.js [hunk 2]
```diff
@@ -1,40 +1,41 @@
 import styled, { css } from "styled-components";
 import { Button } from "antd-mobile";
 import { ROYAL_BLUE, SELAGO } from "../../constants/colors";
-
-/* 
-  Props accepted: 
+import { theme } from "../../constants/theme";
+const { colors } = theme;
```
[the remark this answers is not in the document]
Comment (luxumbra) [reply client/src/components/Button/CustomButton.js:5]: Ah! It was nested destructuring that I was trying to remember! Brilliant, thanks for the review.

[the remark this answers is not in the document]
Comment (luxumbra) [reply client/src/components/Button/CustomButton.js:5]: Is it possible to destructure nested objects in an import? My linter is throwing errors.

File: client/src/components/Button/CustomButton.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import styled, { css } from "styled-components";
 import { Button } from "antd-mobile";
-import { ROYAL_BLUE, SELAGO } from "../../constants/colors";
-import { theme } from "../../constants/theme";
+
+import { theme, mq } from "../../constants/theme";
```
Comment (luxumbra) [inline client/src/components/Button/CustomButton.js:4]: I tried destructuring the `theme` object with `import {theme: { colors } } from "../../constants/colors"` but it caused errors in my linter. Is this possible in imports?

**Comment by luxumbra** [comment]
image
image
image

**Comment by luxumbra** [comment]
I'm not sure how to make the location search work. If someone could give pointers, that'd be amazing! :smile:
Aside from that, the steps are in the correct order and we no longer have the extraneous step 4. If we can get this merged, I can get the rest of the wizards into better shape style wise.

**Comment by luxumbra** [comment]
@shivanshs9 could you check my updates and approve the PR if you're happy with the updates. :pray:

File: client/src/components/Button/CustomButton.js [hunk 4]
```diff
@@ -1,40 +1,41 @@
 import styled, { css } from "styled-components";
 import { Button } from "antd-mobile";
 import { ROYAL_BLUE, SELAGO } from "../../constants/colors";
```
[the remark this answers is not in the document]
Comment (luxumbra) [reply client/src/components/Button/CustomButton.js]: I've now removed the colors import on line 3 but the others are in use.

File: client/src/components/Button/CustomButton.js (hunk 3 above)
Comment (robinv85) [reply client/src/components/Button/CustomButton.js:4]: seems like it's not entirely the same as object destructuring https://stackoverflow.com/questions/41137896/how-to-use-deep-destructuring-on-imports-in-es6-syntax

**Comment by robinv85** [comment]
@luxumbra also I think you asked about the fonts without scss file, there are a lot of ways this can be done, so probably best to pick the best option for performance
in create-react-app docs they mention you can just use "import" and webpack will put it in the bundle https://create-react-app.dev/docs/adding-images-fonts-and-files/ and this is also true for the SVG icons... we will probably need to fine tune the webpack chunks to improve performance as well.

You could also use styled components and use the "createGlobalStyles" method, something like this: 
```
import { createGlobalStyle } from 'styled-components';

import NameOfYourFontWoff from './nameOfYourFont.woff';
import NameOfYourFontWoff2 from './nameOfYourFont.woff2';

export default createGlobalStyle`
    @font-face {
        font-family: 'Font Name';
        src: local('Font Name'), local('FontName'),
        url(${NameOfYourFontWoff2}) format('woff2'),
        url(${NameOfYourFontWoff}) format('woff');
        font-weight: 300;
        font-style: normal;
    }
`;
````

**Comment by luxumbra** [comment]
> also I think you asked about the fonts without scss file, there are a lot of ways this can be done, so probably best to pick the best option for performance

OK cool. This sounds like it should be done in another branch? This PR already has quite a few changes in it.

**Comment by luxumbra** [comment]
Are you happy to merge @robinv85? I'm concerned that the longer we aren't using the theme, it's going to be more work to add it in at a later date.

**Pull request merged**